src/_data/sidenav.yml

```diff
@@ -1,38 +1,65 @@
-- title: Get started
+- header: Get started
+- title: Installation
   children:
     - title: Install Flutter
       permalink: /get-started/install
     - title: Test drive
       permalink: /get-started/test-drive
+
+
+- title: Learn Flutter
+  children:
+    - title: Introduction
+      permalink: /get-started/learn-flutter
+    - title: Dart language overview
+      permalink: https://dart.dev/overview
     - title: Write your first app
       permalink: /get-started/codelab
-    - title: Learn more
-      permalink: /get-started/learn-more
-    - divider
-    - title: Where to get Flutter support
-      permalink: /resources/support
-    - title: From another platform?
-      permalink: /get-started/flutter-for
+    - title: Fundamentals course
```

Comment:
Is the word "course" necessary? I'm worried it might scare some readers away by the long-term nature/time commitment the word conveys.

Perhaps just "Fundamentals", "Learn the fundamentals", "Explore the Fundamentals", etc?